### DISCUSSION TITLE
Send a table in an email

### DISCUSSION THREAD

**Discussion by bastardsandcheats** [ToolJet/ToolJet#5534] [Q&A]
Hi, 
Does anyone have any idea how I would go about having users share a table through an email. 

I can build table like this, and send it with smtp 

```

<table>
   {{Object.values(components.table1.filteredData).map(data => `
   <tr>
      <td>${data.COL_A}</td>
      <td>${data.COL_B}</td>
      <td>${data.COL_C}</td>
      <td>${data.COL_D}</td>
	  `).join('</tr><tr>')
   }}
</table>

```

But then they can only send from whatever address smtp is setup with, it would incredibly useful if I could have that table open in the client's mail app.

**Comment by emidhun** [comment]
Hi @bastardsandcheats,
I think your approach seems to be the right one,
Could you please try something like `currentData` or `currentPageData` , `filteredData`(only if filter is applied).

```
{{Object.values(components.table1.currentData).map(data => `
   <tr>
      <td>${data.id}</td>
      <td>${data.name}</td>
      <td>${data.email}</td>
	  `).join('</tr><tr>')}}
</table>
```
